[Discount] Product guide (#2767)
ibexa/documentation-developer: docs/discounts/discounts_guide.md

---
description: Discounts LTS Update enables reducing prices on products or product categories based on a detailed logic resolution.
month_change: false
editions:
    - lts-update
    - commerce
---

# Discounts product guide

> ## What are Discounts

Reviewer: Not sure if capital letter is needed here (I know that it's a name of the feature, but here it's just an explanation; line 15 - "discounts" without capital as well).
Another option: Discounts feature

```suggestion
## What are discounts
```

They can choose how the discount is calculated and set conditions to decide when their discounts are applied.

The conditions used to limit the applicability of a discount include, for example, rules that check whether:

- the product belongs to a specific category
- the customer belongs to a specific customer group
- minimum purchase amount (total cart value) is met
- minimum purchase quantity (per product) is met

!!! note "Difference between discounts and price rules"

    Unlike flexible and highly configurable discounts, [prices applied to customer groups](prices.md#custom-pricing) cannot have time limits, only apply to specific customer groups, and do not offer flexibility to adjust prices at cart level.

## Availability

Discounts are an opt-in capability available as an [LTS update](editions.md#lts-updates) starting with the v4.6.19 version of [[= product_name_com =]].
To begin using Discounts, you must first [install the required packages and perform initial configuration](install_discounts.md).

## How it works

> The discount feature hooks into the price resolving logic of products, allowing you to modify it before it's displayed to the customers.

Reviewer: ```suggestion
The Discounts feature hooks into the price resolving logic of products, allowing you to modify it before it's displayed to the customers.
```

### Core concepts

#### Discounts

Discounts are reductions in the price of a product, typically implemented as part of a marketing campaign.

Discounts are applied in two places:

> - **Catalog discounts** are activated when browsing the product catalog and do not require any action from the customer to be activated

Reviewer: ```suggestion
- **Product catalog** - catalog discounts are activated when browsing the product catalog and do not require any action from the customer to be activated
```

> - **Cart discounts** can activate when entering the [cart](cart.md), if the right conditions are met. They may also require entering a discount code to be activated

Reviewer: ```suggestion
- **Cart** - cart discounts can activate when entering the [cart](cart.md), if the right conditions are met. They may also require entering a discount code to be activated
```

A shopping cart can have multiple active discounts, but a specific product can only have a single discount applied to it at a time.

> When two or more discounts could be applied to a single product, the system evaluates the following properties to choose the right one:

Reviewer: ```suggestion
When two or more discounts can be applied to a single product, the system evaluates the following properties to choose the right one:
```

The properties are evaluated in the order given above until a single discount is selected.

#### Discount properties

After choosing where the discount applies (catalog or cart), you can choose the discount type:

- **Fixed amount** - where a specified amount of money, for example, 5 Euro, is deducted from the base price of the product
- **Percentage** - where a specified percentage, for example, 10%, is used to calculate the deducted amount from the product's base price

Discounts are translatable and you can limit them to specific [regions](pim_guide.md#regions) or a single currency.
They can be permanent or be active only in a specified time frame.
Regardless of the specified dates, you can disable a discount at any time to prevent customers from using it.

The discount data is split into two parts:

- name and description add internal information for the store managers
- promotion information add additional information displayed to the customers

#### Target groups

With discounts, you can target your entire customer base or only a subset of it belonging to specified [customer groups](customer_groups.md).

#### Product selection

> All products, including [product variants](pim_guide.md#product-variants), can be selected when creating a discount. You can also limit this choice to a subset of products:

Reviewer: ```suggestion
All products, including [product variants](pim_guide.md#product-variants), can be selected when creating a discount.
You can also limit this choice to a subset of products:
```

Discounts are reductions in the price of a product, typically implemented as part of a marketing campaign.

Discounts are applied in two places:

- **catalog** discounts are activated when browsing the product catalog and do not require any action from the customer to be activated
- **cart discounts** are activated when browsing the [cart](cart.md) and may require entering a discount code to be activated

A shopping cart can have multiple active discounts, but a specific product can only have a single discount applied to it at a time.

When two or more discounts could be applied to a single product, the system evaluates the following properties to choose the right one:

- discount activation place (cart discounts rank higher over catalog discounts)
- discount priority (higher priority ranks higher)
- creation date (newer discounts ranks higher)

The properties are evaluated in the order given above until a single discount is selected.

#### Discount properties

After choosing where the discount applies (catalog or cart), you can choose the discount type:

- **Fixed amount** - where a specified amount of money, for example, 5 Euro, is deducted from the base price of the product

> - **Percentage** - where a specified percentage, for example, 10%, is used to calculate the deducted amount from the product

Reviewer 2: ```suggestion
- **Percentage** - where a specified percentage, for example, 10%, is used to calculate the deducted amount from the product's base price
```

> Discounts are translatable and are valid for specific [regions](pim_guide.md#regions) and currencies.

Reviewer 2: Maybe it's worth to mention that only one currency can be selected.
